bundles/org.openhab.binding.dbquery/README.md

```diff
@@ -0,0 +1,213 @@
+| scalarResult | false    | true     | If query always returns a single value or not                         |
+| scalarColumn | false    |          | In case of multiple columns, it indicates which to use for scalarResult|
+
+that are described further in the following subsections
+
+##### query  
+
+The query the items represents in the native language of your database:
+
+ - Flux for `influxdb2`
+ 
+#### hasParameters
+
+If `hasParameters=true` you can use parameters in the query string that can be dynamically set with the `setQueryParameters` action.
+ 
+ For InfluxDB use the `${paramName}` syntax for each parameter, and keep in mind that the values from that parameters must be from a trusted source as current
+ parameter substitution is subject to query injection attacks.
+ 
+#### timeout
+
+A time-out in seconds to wait for the query result, if it's exceeded, result will be discarded and the addon will do its best effort to cancel the query.
+Currently it's ignored and it will be implemented in a future version.
+
+#### scalarResult 
+
+If `true` the query is expected to return a single scalar value that will be available to `result` channels as string, number, boolean,...
+If the query can return several rows and/or several columns per row then it needs to be set to `false` and the result can be retrieved in `resultString`
+channel as JSON or using the `getLastQueryResult` action.   
+
+#### scalarColumn
+
+In case `scalarResult` is `true` and the select returns multiple columns you can use that parameter to choose which column to use to extract the result.
+
+## Channels
+
+Query items offer the following channels to be able to query / bind them to items:
+
+| Channel Type ID | Item Type | Description                                                                                                                        |
+|-----------------|-----------|------------------------------------------------------------------------------------------------------------------------------------|
+| execute         | Switch    | Send `ON` to execute the query manually. It also indicates if query is currently running (`ON`) or not running (`OFF`)          |
+| resultString    | String    | Result of last executed query as an String |
+| resultNumber    | Number    | Result of last executed query as a Number, query must have `scalarResult=true` |
+| resultDateTime  | DateTime  | Result of last executed query as a DateTime, query must have `scalarResult=true` |
+| resultContact   | Contact   | Result of last executed query as Contact, query must have `scalarResult=true` |
+| resultSwitch    | Switch    | Result of last executed query as Switch, query must have `scalarResult=true` |
+| parameters      | String    | Contains parameters of last executed query as JSON|
+| correct         | Switch    | `ON` if the result of last executed query was correct |
```

Comment:
Can you make this clearer as I am not sure how this is different to resultSwitch channel? Do you mean `correct`, or do you mean true like a boolean?

Reply:
So do I understand it if I suggest this wording?
```
`ON` if the last executed query completed successfully, `OFF` if the query failed.
```
I feel the description does not explain what the channel is for when someone is new to the binding, perhaps I am a slow learner :)

Reply:
Thanks, sounds better